chia/consensus/block_body_validation.py

```diff
@@ -287,37 +288,34 @@ async def validate_block_body(
                 return Err.DOUBLE_SPEND, None
 
         # 15. Check if removals exist and were not previously spent. (unspent_db + diff_store + this_block)
+        # The fork point is the last block in common between the peak chain and the chain of `block`
         if peak is None or height == 0:
             fork_h: int = -1
         elif fork_point_with_peak is not None:
             fork_h = fork_point_with_peak
         else:
             fork_h = find_fork_point_in_chain(blocks, peak, blocks.block_record(block.prev_header_hash))
 
-        if fork_h == -1:
-            coin_store_reorg_height = -1
-        else:
-            last_block_in_common = await blocks.get_block_record_from_db(blocks.height_to_hash(uint32(fork_h)))
-            assert last_block_in_common is not None
-            coin_store_reorg_height = last_block_in_common.height
```

Comment:
makes sense. this is looking up the block by height, then looking at its height. we knew that already..